AdyenTests/Card Tests/CardComponentTests.swift

```diff
@@ -247,7 +247,7 @@ class CardComponentTests: XCTestCase {
         let method = CardPaymentMethod(type: "bcmc", name: "Test name", fundingSource: .credit, brands: ["any_test_brand_name"])
         let sut = CardComponent(paymentMethod: method,
                                 clientKey: "test_client_key")
-        sut.showsLargeTitle = false
+        sut._showsLargeTitle = false
```

Comment:
I think we can delete those :)

Reply:
don't think so, they are still needed, to setup the test preconditions.